unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreenV2/Tests/TestsEditMode/HintViewManagerShould.cs

```diff
@@ -57,7 +65,11 @@ public void TearDown()
         public async Task StartAndStopHintsCarousel()
         {
             // Arrange
-            var loadingScreenHintsController = new LoadingScreenHintsController(hintRequestService);
+            LoadingScreenView loadingScreenView = GameObject.Instantiate(Resources.Load<GameObject>(LOADING_SCREEN_ASSET)).GetComponent<LoadingScreenView>();
+            var loadingScreenHintsController = new LoadingScreenHintsController(hintRequestService, loadingScreenView, addressableProvider);
+            loadingScreenView.ToggleLoadingScreenV2(true);
+            var loadingScreenV2ProxyPlugin = new LoadingScreenV2ProxyPlugin();
+            loadingScreenHintsController = await loadingScreenV2ProxyPlugin.InitializeAsync(loadingScreenView, addressableProvider, cancellationToken);
```

Comment:
We need to rethink the test suite for this stuff. This is not a unit test, it's basically creating the whole environment to only test the carousel. It's too much.

If you are testing a view here you shouldnt need a controller and never a plugin. It should be way more simpler because views shouldnt have many dependencies to mock.